src/status_im/contexts/wallet/collectible/tabs/about/view.cljs

```diff
@@ -0,0 +1,47 @@
+(ns status-im.contexts.wallet.collectible.tabs.about.view
+  (:require [quo.core :as quo]
+            [quo.theme]
+            [react-native.core :as rn]
+            [status-im.contexts.wallet.collectible.tabs.about.style :as style]
+            [utils.re-frame :as rf]))
+
+(def link-cards
+  [{:title               "BAYC"
```

Comment:
this data seems like mock data, can we move it to the temp folder so it's easier to realise this?

Reply:
or is it possible to get real data already @vkjr ??

Reply:
The ticket is to implement the UI but if you think I can get the real data, then I will 👍

Reply:
yeah if it is not too much extra work to do so, e.g the subs etc are mostly already existing then perhaps it's better to do in this pr.

Reply:
@mmilad75, @J-Son89, I don't see any links data coming from `status-go` side, so probably it requires additional work on backend side. I would go with temp data for now